package.json

```diff
@@ -16,7 +16,8 @@
     "webdriver": "webdriver-manager update && webdriver-manager start",
     "wdio-local": "rm -rf allure-results allure-report && grunt e2e-env-setup && wdio run ./tests/wdio.conf.js",
     "e2e-integration": "mocha --config tests/integration/.mocharc.js ",
-    "wdio": "wdio run ./tests/wdio.conf.js"
+    "wdio": "wdio run ./tests/wdio.conf.js",
```

Comment:
Probably a good idea to start creating tasks to get called by other tasks. 

```
    "wdio-setup": "rm -rf allure-results allure-report && grunt e2e-env-setup", 
    "wdio-local": "npm run wdio-setup && grunt e2e-env-setup && wdio run ./tests/wdio.conf.js",
    "wdio-standard": "npm run wdio-setup && wdio run ./tests/wdio-standard.conf.js"
```

Reply:
Done